__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const request = require("supertest");
+const app = require("../app.js");
+const seed = require("../db/seeds/seed.js");
+const {
```

Comment:
Rarther than destructuring the items off the object here it may be slightly cleaner to import the whole data object and pass this into your seed function as a single item however this is very readable to its completely up to you 👍